tampermonkey/chatwidget.user.js

```diff
@@ -11,7 +11,7 @@
 (function() {
     'use strict';
 
-    const widgetURL = "https://uxwill.github.io/chatinc/";
+    const widgetURL = "https://mechanical-ink.github.io/chatinc-1/";
```

Comment:
I doubt that that repo still exists. I created a new fork that is just `chatinc`. Let me deploy that one to Github pages. That means that this URL will be 
```suggestion
    const widgetURL = "https://mechanical-ink.github.io/chatinc/";
```

Reply:
Done, accessible here: https://mechanical-ink.github.io/chatinc/